gap/examples.gi

```diff
@@ -370,3 +370,47 @@ GeneralisedPetersenGraphCons);
 
 InstallMethod(GeneralisedPetersenGraph, "for integer, integer", [IsInt, IsInt],
 {n, k} -> GeneralisedPetersenGraphCons(IsImmutableDigraph, n, k));
+
+InstallMethod(TadpoleDigraphCons, "for IsMutableDigraph and two integers",
+[IsMutableDigraph, IsInt, IsInt],
+function(filt, m, n)
+  local i, j, tail, graph;
+  if (m < 3) then
```

Comment:
Brackets aren't required in this situation in GAP, and we typically avoid them unless we really need them for clarity.
```suggestion
  if m < 3 then
```